plutarch-test/plutarch-extra/Plutarch/ListUtilsSpec.hs

```diff
@@ -0,0 +1,36 @@
+module Plutarch.ListUtilsSpec (spec) where
+
+import Test.Syd
+import Test.Syd.Hedgehog ()
+
+import Plutarch.Prelude
+import Plutarch.Test
+
+import Hedgehog (Property)
+
+import Plutarch.Test.Property.Gen (genList, integerGen)
+import Plutarch.Test.Property.Util (haskPlutEquiv, marshal, viaPEq)
+
+import Plutarch.ListUtils (pcheckSorted, preverse)
+
+spec :: Spec
+spec = do
+  describe "extra.listutils" $ do
+    describe "properties" $ do
+      describe "reverse" $ do
+        it "plutarch level reversing behaves like haskell level reversing" reverseTest
+    describe "goldens" . pgoldenSpec $ do
```

Comment:
`describe "goldens"` should be removed (it is implicitly injected by `pgoldenSpec`).